ide/project.dependency/nbproject/project.properties

```
@@ -16,7 +16,8 @@
 # under the License.
 
 is.autoload=true
-javac.source=11
-javac.target=11
+javac.source=17
+javac.target=17
```

Suggest (assuming it will compile - `javac.release` will use the `--release 17` javac option, which compiles against the specific API from JDK 17): 
```suggestion
javac.release=17
```